vm-setup/roles/libvirt/tasks/vm_setup_tasks.yml

```diff
@@ -133,8 +133,14 @@
 # task *must* occur after the above vm tasks, because if
 # `vm_nodes` is defined the template depends on the
 # `node_mac_map` variable.
-- name: Write ironic node json files
+- name: Write metal3 node json files
   template:
-    src: ../templates/ironic_nodes.json.j2
+    src: ../templates/metal3_nodes.json.j2
     dest: "{{ nodes_file }}"
     force: no
+
+- name: Write ironic-compatible node json files
+  template:
+    src: ../templates/ironic_nodes.json.j2
+    dest: "{{ working_dir }}/ironic_nodes.json"
```

Comment:
We should make this a variable or this will overwrite nodes_file in the case where it's set to ironic_nodes.json and also dev-scripts writes out the existing file to a per-cluster directory, so we'll want to do the same for this file.